docs/superpowers/plans/2026-05-27-component-query-rewrite-fixes.md

```
@@ -0,0 +1,275 @@
+  #include <mysql/components/services/udf_registration.h>
+  #include <mysql/components/services/dynamic_loader_service_notification.h>
+  #include "mysql/components/util/event_tracking/event_tracking_parse_consumer_helper.h"
+  #include "myvector.h"
+  #include "myvector_binlog_service.h"
+  #include "myvector_udf_service.h"
+  ```
+
+- [ ] **Step 3: Add the notify function and service implementation before the PROVIDES block**
+
+  The current `src/component_src/myvector_component.cc` at line 59 reads:
+  ```cpp
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)